Content.Server/Weapons/Melee/Shield/EnergyShieldSystem.cs

```diff
@@ -0,0 +1,114 @@
+using Content.Server.CombatMode.Disarm;
+using Content.Server.Kitchen.Components;
+using Content.Shared.Interaction;
+using Content.Shared.Interaction.Events;
+using Content.Shared.Item;
+using Content.Shared.Temperature;
+using Content.Shared.Toggleable;
+using Content.Shared.Tools.Components;
+using Robust.Shared.Player;
+
+namespace Content.Server.Weapons.Melee.EnergyShield;
+
+public sealed class EnergyShieldSystem : EntitySystem
```

Comment:
Same thing as the component, if all of the events and dependencies are in shared then this should be as well.

Reply:
done